lightning/src/chain/keysinterface.rs

```diff
@@ -98,11 +98,14 @@ pub enum SpendableOutputDescriptor {
 		to_self_delay: u16,
 		/// The output which is referenced by the given outpoint
 		output: TxOut,
-		/// The channel keys state used to proceed to derivation of signing key. Must
-		/// be pass to KeysInterface::derive_channel_keys.
-		key_derivation_params: (u64, u64),
 		/// The revocation_pubkey used to derive witnessScript
-		revocation_pubkey: PublicKey
+		revocation_pubkey: PublicKey,
+		/// Arbitrary identification information returned by a call to
+		/// `ChannelKeys::key_derivation_params()`. This may be useful in re-deriving keys used in
+		/// the channel to spend the output.
+		key_derivation_params: (u64, u64),
+		/// The value of the channel which this transactions spends.
```

Comment:
I don't think that's a direct spend, it's more likely a grand-children from the funding transaction. Maybe "The value of the channel which this transaction is part of the onchain graph" ?